README.md

```diff
@@ -67,8 +67,16 @@ Note: integration tests are currently disabled since the backend server is curre
 
 ### Accessibility Tests
 
-1. Start the application by running `npm start` or `npm run dev`
-2. In another terminal, run the tests with this command `pa11y-ci`
+#### Locally
+1. Add your local end point to the urls in the .pa11yci file (ex. https://localhost:3000/job-role)
```

Comment:
I don't like this - it should be easy for devs to run this test locally vs running against an environment. I fill forget where I am testing. We should parameterize the urls - I don't know how that looks, but it's really important to automate